modules/monitoring-table-of-remote-write-metrics.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/troubleshooting-monitoring-issues.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="table-of-remote-write-metrics_{context}"]
+= Table of remote write metrics
+
+The following table contains remote write and remote write-adjacent metrics with further description to help during troubleshooting of remote write storage.
+
+[options="header"]
+|===
+| Metric | Description
+| `prometheus_remote_storage_highest_timestamp_in_seconds` | Shows the newest timestamp that Prometheus stored in the write-ahead log (WAL) for any sample.
+| `prometheus_remote_storage_queue_highest_sent_timestamp_seconds` | Shows the newest timestamp that remote write successfully sent.
```

Comment:
```suggestion
| `prometheus_remote_storage_queue_highest_sent_timestamp_seconds` | Shows the newest timestamp that remote write successfully sent by this queue.
```
based on
`# HELP prometheus_remote_storage_queue_highest_sent_timestamp_seconds Timestamp from a WAL sample, the highest timestamp successfully sent by this queue, in seconds since epoch. Initialized to 0 when no data has been sent yet.`